### DISCUSSION TITLE
Send a toast from a child iFrame to be displayed in the parent

### DISCUSSION THREAD

**Discussion by gCardinal** [fkhadra/react-toastify#722] [Q&A]
Imagine I have a React app that serves as a makeshift microfrontend, loading applications in an iFrame. I would like for this top-level React application to be able to provide a method for child applications in the iFrame to display toasts.

The goal here is the child application does not need to concern itself on where the toast is supposed to be displayed and what it should look like.

I can easily provide the `toast()` method on the child iFrame's `window` object, but calling this method won't display a toast. I also tried calling the method by exposing it to the parent's `window` object, still no luck.

I'm guessing it's not finding the toast container, but I'm not sure since in theory, calling it from the parent's context should work? I must be missing something silly...

**Comment by bdresdner** [comment]
Hi @gCardinal - I have a similar scenario.  Did you ever find a way to do this?

**Reply by gCardinal** [reply]
Oh, I wish I could tell you, but this was in a contract job a while ago and I do not have access to the codebase anymore and I can't remember if I had solved the issue. I'm assuming the answer is no, I like to believe I would have updated this question if I had.

**Reply by bdresdner** [reply]
No worries!  Thanks anyway.
